test/deferred.js

```diff
@@ -1,4 +0,0 @@
-import $ from 'jquery';
-
-export default $.Deferred();
```

Comment:
We should keep this text else stylelint fixer does convert `$.Deferred` to `$.deferred` which actually targets a not existing jquery method:

https://github.com/massiveart/eslint-config-ma/pull/2

Reply:
@alexander-schranz Done!